### PR TITLE
Add SSH instructions for Windows/MacOS

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: website/docs/ssh/README.md [hunk 1]
```diff
@@ -53,13 +53,47 @@ on your computer that you'd like to use.
...
+start .
+```
+
+c. Hit **Enter**
+
+d. This will open the folder where your `SSH` key was saved.
+
+e. Open the `id_rsa.pub` on a text editor such as Notepad.
+
+f. Copy the entire content of the file (including ssh-rsa) and paste in the website (Step 7)
```
Comment (Hackerman342) [inline website/docs/ssh/README.md:84]: But a better option would be to combine the two sections (display & copy) as a single section (copy)
```suggestion
f. Copy the entire content of the file (including ssh-rsa) and paste in the website (Step 11)
```

File: website/docs/ssh/README.md [hunk 2]
```diff
@@ -88,7 +122,77 @@ import Tabs from '@theme/Tabs';
...
+
+c. Hit **Enter**
+
+d. This will open the folder where you need to save your configurations.
+
+e. Create a new file and name it config.
+
+**Hint**: Create a copy of your ssh_rsa.pub, and rename it to config;
+but make sure that it doesn't have an extension (e.g., ".pub").
+
```
Comment (Hackerman342) [inline website/docs/ssh/README.md:173]: Not a big fan of this hint
```suggestion
```

**Review by Hackerman342 — APPROVED** [review]
LGTM

File: website/docs/ssh/README.md (hunk 2 above)
Comment (guilhermedemouraa) [reply website/docs/ssh/README.md]: Proposed alternative in [34928e6](https://github.com/farm-ng/amiga-dev-kit/pull/216/commits/34928e6fd63611cac9da8019e184409b50f789cf)

**Pull request merged**